decdat/decompiler.py

```diff
@@ -75,6 +74,10 @@ def decompile_parameter(self, symbol_type=None):
             return self.decompile_call()
         elif op_type in (TokenEnum.BINARY_OP, TokenEnum.UNARY_OP):
             return self.decompile_operation()
+        elif op_type == TokenEnum.JUMP:
```

Comment:
Could you provide here an example of Daedalus script and corresponding .DAT file that I could relate to these changes?

Reply:
Dat file is an chronicles of myrtana one. For obvious reasons I don't have daedalus source files.